driver-sync/src/test/functional/com/mongodb/client/AbstractUnifiedTest.java

```diff
@@ -386,9 +439,73 @@ public Void execute() {
                             }, transactionOptions);
                         }
                     } else if (operationName.equals("targetedFailPoint")) {
-                        assertTrue(failPoint == null);
+                        assertNull(failPoint);
                         failPoint = new TargetedFailPoint(operation);
                         failPoint.executeFailPoint();
+                    } else if (operationName.equals("configureFailPoint")) {
```

Comment:
This supports all the new operations required by the new SDAM integration test runner.